framework/resource/metricsresource/resource_test.go

```diff
@@ -17,9 +17,10 @@ func Test_MetricsResource_ProcessDelete_ResourceOrder(t *testing.T) {
 		tr,
 	}
 
-	config := DefaultWrapConfig()
-	config.Name = t.Name()
-	wrapped, err := Wrap(rs, config)
+	c := WrapConfig{
+		Name: t.Name(),
+	}
```

Comment:
I would like to keep it this way. 

```
c := Config{}

c.Name = foo

...
```

No braces, indentations, no commas, no colons, no additional blocks. Just plain and straight assignments.

Reply:
I saw in several places you use this style. I'm against it. Struct initialization is very idiomatic.
I didn't merge https://github.com/giantswarm/fmt/pull/4 yet. We can discuss here with rest of dev team. If you can convince them, I'll follow.